### DISCUSSION TITLE
Adding support for 3.14 (WIP)

### DISCUSSION THREAD

**Discussion by clin1234** [PyO3/maturin#2402] [Ideas]
Here's my fork so far: https://github.com/clin1234/maturin

I think `pyo3` needs to also support 3.14 for the tests to pass...

**Comment by messense** [comment]
Thanks for the effort, but we are not going to add support for Python alpha releases, it will be added after 3.14 beta1 release, which is expected to be released on [2025-05-06](https://peps.python.org/pep-0745/#schedule).

**Reply by clin1234** [reply]
Just trying to make maturin work when beta1 comes out
